server/e2e/publicapi_test.go

```diff
@@ -270,12 +270,13 @@ func publicAPISeeder(ctx context.Context, r *repo.Container) error {
 		FileName("aaa.zip").UUID(publicAPIAssetUUID).MustBuild()
 	af := asset.NewFile().Name("bbb.txt").Path("aaa/bbb.txt").Build()
 
+	fId := id.NewFieldID()
```

Comment:
```suggestion
	fid := id.NewFieldID()
```

In golang, `Id` is not used as a variable name and a field name. Use `id` or `ID` instead.

See https://github.com/golang/go/wiki/CodeReviewComments#initialisms

> Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".
> 
> This rule also applies to "ID" when it is short for "identifier" (which is pretty much all cases when it's not the "id" as in "ego", "superego"), so write "appID" instead of "appId".
>
> Code generated by the protocol buffer compiler is exempt from this rule. Human-written code is held to a higher standard than machine-written code.

e.g.

```go
type Hoge struct {
  aId string // ng
  aID string // ok
  aid string // ok
}
```

Reply:
Okay so it is either fid or fID.
Done!

Reply:
Okay, so fid and fID are acceptable but fId is not, understood._
Done!